exporter/datadogexporter/internal/metrics/utils.go

```diff
@@ -0,0 +1,86 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
`utils` was such an unfortunate name. It is many times considered a smell in Go. It is very ambiguous. It's unclear what it means. Can we maybe find a better name for it? `series.go`? I don't know. In the end, if we can't find a better name, "utils" is ok. But maybe we can try :)

Reply:
Agreed, when I first started programming in Go, I got the same suggestion that `utils` is not a recommended name in Go... I think `series` makes sense since these convenient functions are used to build metric series, I've made the name changes.